azurerm/resource_arm_operational_insight_workspace.go

```diff
@@ -0,0 +1,207 @@
+package azurerm
```

Comment:
(as above) - could we rename this to `resource_arm_log_analysis_workspace.go`

Reply:
Done!